### DISCUSSION TITLE
Where are CSS styles?

### DISCUSSION THREAD

**Discussion by piw-piw** [TurboWarp/packager#510] [General]
Where are CSS styles (like text cloud or input) which are loaded into a project?

**Comment by GarboMuffin** [comment]
> text cloud

if you mean the text bubbles for blocks like say/think, that's not CSS

> input

https://github.com/TurboWarp/packager/blob/master/src/scaffolding/style.css ctrl+f "question"

All classes in this file are prefixed with `sc-` on the built website

**Comment by piw-piw** [comment]
I tried to set accent color to green, but it doesn't work.
![image](https://user-images.githubusercontent.com/76477070/187146411-eb7b9591-8749-4ad1-8c63-541ee2ec43c1.png)

I changed line `scaffolding.setAccentColor("#00ff00");` in the file and it works.

**Reply by GarboMuffin** [reply]
setAccentColor overrides the CSS variables. You should use the dedicated option for accent color instead of custom CSS or use `!important` if there's a good reason to use CSS.
